src/main/kotlin/uk/gov/gdx/datashare/services/AdminService.kt

```diff
@@ -12,12 +12,14 @@ class AdminService(
   private val cognitoService: CognitoService,
   private val acquirersService: AcquirersService,
   private val suppliersService: SuppliersService,
+  private val adminActionAlertsService: AdminActionAlertsService,
 ) {
   fun createCognitoClient(cognitoClientRequest: CognitoClientRequest) =
     cognitoService.createUserPoolClient(cognitoClientRequest)
 
   @Transactional
   fun createAcquirer(createAcquirerRequest: CreateAcquirerRequest): CognitoClientResponse? {
+    adminActionAlertsService.noticeAction(AdminAction("Create Acquirer", createAcquirerRequest))
```

Comment:
Would we want to alert on success or attempt? I fear that if this fails we would still get the alert saying that it had been created.